.github/workflows/test-build-spack.yaml

```diff
@@ -0,0 +1,80 @@
+name: Build ESMF Using Spack
+
+on:
+  workflow_dispatch:
+  push:
+  schedule:
+    - cron: '0 0 * * MON'
+    - cron: '0 0 * * FRI'
+
+jobs:
+  latest-stable:
```

Comment:
Probably need a different name here, e.g. `spack-test`, since we don't specifically test the latest stable only.